src/content/docs/Products/SplashKit/Splashkit Expansion/Documentation/onboarding-guide.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: SplashKit Expansion Onboarding Guide
+sidebar:
+  label: SplashKit Expansion Onboarding Guide
+  order: 1
+---
+
+## Introduction
+
+This guide will cover all the steps required to get contributing to the splashkit-core repository.
+Feel free to skip steps which you have already completed or are familiar with.
+
+## Installing WSL
+
+WSL is a built-in Linux distribution virtual machine for Windows. splashkit-core will be installed
+to the Linux distribution. The official Microsoft documentation provides high-quality instructions
+on how to install WSL: [Install WSL](https://learn.microsoft.com/en-us/windows/wsl/install). This
+guide has been tested with the default Ubuntu distribution, but others may also work.
+
+### Installing Visual Studio Code
+
+Once WSL has been installed to Windows, Visual Studio Code needs to be installed to WSL.
+The Microsoft documentation is again the best reference for this process:
+[Install VS Code](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-vscode).
+
+### Installing Git
+
+Now Git must be installed to WSL. Follow the Microsoft installation instructions:
+[Install Git](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-git).
```

Comment:
Here you could also add a link to - https://education.github.com/git-cheat-sheet-education.pdf